app/helpers/application_helper.rb

```diff
@@ -7,6 +7,11 @@ def current_user_customer?
     current_user.present? && current_user.customer?
   end
 
+  def current_user_own_book?(book)
+    current_user_admin? ||
+        (current_user_customer? && current_user.books.find_by(id: book.id) && book.pdf_file.attached?)
+  end

```

Comment:
I think it should be like this
(current_user_admin?  && **book.pdf_file.attached?** )||
        (current_user_customer? && current_user.books.find_by(id: book.id) && book.pdf_file.attached?)

Reply:
fair point. Will update it.